src/components/molecules/AnnouncementMessage/AnnouncementMessage.tsx

```diff
@@ -33,16 +33,22 @@ export const AnnouncementMessage: React.FC<AnnouncementMessageProps> = ({
 
   return (
     <div
+      aria-labelledby="announcement-container-message"
+      role="dialog"
       className={classNames("announcement-container", {
         centered: !isExpanded,
       })}
     >
-      <div className="announcement-message">
+      <div className="announcement-message" id="announcement-container-message">
```

Comment:
Any reason to add id?

Reply:
I use aria-lableby to target this element.

Reply:
@oleksii-front could you explain it, please?

Reply:
@mike-lvov go to example 4 please https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

Reply:
This is one of the very few sotuations where we actually need to use an id in react/etc.

In using an id though, we need to ensure it’s unique within the rendered application. I haven’t looked at the code here, but that’s something we need to ensure